package.json

```diff
@@ -39,13 +39,16 @@
     "nuke": "yarn run nuke:packages && yarn run nuke:docker",
     "nuke:packages": "yarn run restore",
     "nuke:docker": "lerna run --stream dev:stack:nuke",
-    "clean": "lerna clean -y",
+    "clean": "lerna clean -y && echo Cleaning top level node modules 🧹 && rm -rf ./node_modules && echo Done! 🚀",
```

Comment:
How come this is required?

Reply:
Makes sense, I don't mind it staying in this PR. I like to keep the barrier to small improvements like this as low as possible. :)